subsys/bluetooth/controller/hci/hci.c

```diff
@@ -2975,6 +3037,44 @@ void hci_evt_encode(struct radio_pdu_node_rx *node_rx, struct net_buf *buf)
 	}
 }
 
+u8_t hci_evt_concat(struct radio_pdu_node_rx **node_rx, struct net_buf *buf)
+{
+	u8_t node_cnt = 0;
+   struct radio_pdu_node_rx *node_rx_slice[HCI_MAX_NR_OF_CONCAT_MSG] = {NULL};
+
+	while(node_rx[node_cnt] && (node_cnt < HCI_MAX_NR_OF_CONCAT_MSG))
+	{
```

Comment:
Add `node_rx_slice[node_cnt] = node_rx[node_cnt]` here, then you can remove the `memcpy` below